src/main/scala/com/activegrid/model/GraphDBExecutor.scala

```diff
@@ -0,0 +1,93 @@
+package com.activegrid.model
+
+import com.typesafe.scalalogging.Logger
+import eu.fakod.neo4jscala.{EmbeddedGraphDatabaseServiceProvider, Neo4jWrapper}
+import org.neo4j.graphdb.Node
+import org.slf4j.LoggerFactory
+
+/**
+  * Created by sampathr on 22/9/16.
+  */
+object GraphDBExecutor extends Neo4jWrapper with EmbeddedGraphDatabaseServiceProvider {
+  val logger = Logger(LoggerFactory.getLogger(getClass.getName))
+  val V_ID = "V_ID"
+
+  def neo4jStoreDir = "./graphdb/activegrid"
+
+  def persistEntity[T <: BaseEntity : Manifest](entity: T, label: String): Option[T] = {
+    withTx { neo =>
+      val node = createNode(entity, label)(neo)
+    }
+    return Some(entity)
+  }
+
+  def findNodeById(id: Long): Node = withTx { neo =>
+    getNodeById(id)(neo)
+  }
+
+  def getProperties(node: Node, keys: String*): Map[String, Any] = withTx { neo =>
+    val map: scala.collection.mutable.Map[String, AnyRef] = scala.collection.mutable.Map()
+    keys.foreach(key => {
+      map += ((key, node.getProperty(key)))
+    })
+    map.toMap
+  }
+
+  def createGraphNode[T <: BaseEntity : Manifest](entity: T, label: String): Option[Node] =
+    withTx { neo =>
+      val node = createNode(label)(neo)
+      node.setProperty("", "")
```

Comment:
Need to remove it used it earlier now I am not using it and thought of keeping it as placeholder but will remove unused methods